frontend/src/components/pages/student/StudentRouting.tsx

```diff
@@ -61,12 +61,14 @@ const StudentRouting = (): React.ReactElement => {
             exact
             path={Routes.ASSESSMENT_SUMMARY_PAGE}
             roles={["Student"]}
+            title="Assessment Summary"
           />
           <PrivateRoute
             component={WriteAssessmentPage}
             exact
             path={Routes.WRITE_ASSESSMENT_PAGE}
             roles={["Student"]}
+            title="Write Assessment"
```

Comment:
why do we need both this line and the dynamic config in `WriteAssessmentPage`? (same thing with assessment summary)